environment.yml

```diff
@@ -21,7 +22,7 @@ dependencies:
 - glib=2.53.6
 - gst-plugins-base=1.12.2
 - gstreamer=1.12.2
-- h5py=2.7.1
+- h5py=2.7.0
```

Comment:
why do we need an older version of h5py?

Reply:
oups sorry forgot to change this again... I was debugging and tried if my issue comes from here